packages/lexical/src/LexicalNode.ts

```diff
@@ -185,6 +188,53 @@ export class LexicalNode {
   __prev: null | NodeKey;
   /** @internal */
   __next: null | NodeKey;
+  /**
+   * Don't use this directly, use `this.getClasses()` and `this.mutateClasses()` instead
+   * @internal
+   */
+  __classes?: ReadonlyClasses;
```

Comment:
```suggestion
  __classes: ReadonlyClasses;
```

This can be initialized to an empty object in the constructor, and then all other branching based on whether it's undefined or not can be eliminated.